apps/etl/transform/sources/handler.py

```diff
@@ -83,12 +84,13 @@ def handle_transformation(cls, extraction_id: int):
         if not extraction_obj.resp_data:
             logger.info("Transformation ended because there is no data")
             return
-
+        trace_id = get_trace_id(extraction_obj)
         transform_obj = Transform.objects.create(
             extraction=extraction_obj,
-            trace_id=get_trace_id(extraction_obj),
+            trace_id=trace_id,
         )
 
+        SentryTag.set_tags({SentryTag.Tag.SOURCE: extraction_obj.source, SentryTag.Tag.TRACE_ID: extraction_obj.trace_id})
```

Comment:
Let's add a NOTE to move this to the `__init__` after transformer is refactored (when transformer_id is used instead of extraction_id)